renovate.json

```diff
@@ -11,7 +11,6 @@
       "automerge": true
     }
   ],
-  "enabledManagers": ["regex"],
```

Comment:
@gbenhaim are we ok with having all managers enabled? I was not sure if we intentionally enabled just regex.

Reply:
Never mind, for now, I enabled just regex and kustomize, and we can figure out if we want others later.